sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/ChatThreadClient.java

```diff
@@ -52,27 +51,27 @@ public String getChatThreadId() {
     }
 
     /**
-     * Updates a thread's properties.
+     * Updates a thread's topic.
      *
-     * @param options Options for updating a chat thread.
+     * @param topic The new topic.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public void updateChatThread(UpdateChatThreadOptions options) {
+    public void updateTopic(String topic) {
 
-        this.client.updateChatThread(options).block();
+        this.client.updateTopic(topic).block();
```

Comment:
Is "updateTopic" an existing function? I'd surprised if it is. If it is not, how can this compile?